indico/modules/rb/client/styles/responsive.scss

```diff
@@ -0,0 +1,11 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+
+$tablet-width: 768px;
```

Comment:
Out of curiosity, where did you get those values from?

Reply:
I'm not sure, from [semantic-ui](https://semantic-ui.com/elements/container.html) I think, they're kind of _standard_ if you do a quick search (responsive breakpoints), for instance Bootstrap uses the same. Probably @pferreir knows better as he added those.